yt_dlp/extractor/matchtv.py

```diff
@@ -1,48 +1,27 @@
-import random
-
 from .common import InfoExtractor
-from ..utils import xpath_text
-
 
```

Comment:
You are missing a new line after your imports.
```suggestion


```
You can also run ruff locally to automatically fix these things.